adagrams/game.py

```diff
@@ -1,11 +1,114 @@
+import random
+
 def draw_letters():
-    pass
+
+    letter_probability = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+    }
+    
+    # Create pool of letters with frequency as outlined in letter_probability dict
+    pool_of_letters = []
+    for letter in letter_probability:
+        pool_of_letters.extend([letter] * letter_probability[letter])
```

Comment:
Nicely done! Creating a big list to contain all the possible values in their distribution/limit, will ensure the randomness adheres to the distribution as well as constrain the amount of letters used.